src/mlpack/core/dists/discrete_distribution.hpp

```diff
@@ -219,6 +243,25 @@ class DiscreteDistribution
   std::vector<arma::vec> probabilities;
 };
 
+/**
+* Calculates the Discrete log probability function for each
```

Comment:
These are really tiny style issues but the `*` doesn't match the `/**`.  We should aim for

```
/**
 * Comments.
 */
```

not

```
/**
* Comments.
*/
```

Sorry to be so pedantic.  This also happens in the declaration of `LogProbability()` in `gaussian_distribution.hpp`.

Reply:
@rcurtin , I think I have changed the style at both the place. Could you merge this? I will add those functions to all other distributions, once it gets merged.

Thanks